odl_video/settings.py

```diff
@@ -518,3 +519,7 @@ def get_all_config_keys():
         'odl_video.middleware.QueryStringFeatureFlagMiddleware',
         'odl_video.middleware.CookieFeatureFlagMiddleware',
     )
+
+
+# MySQL password, only required for importing TechTV videos.
+MYSQL_PASSWORD = get_string("MYSQL_PASSWORD", "")
```

Comment:
Instead of having this as a setting can you use `os.getenv` since we are only using it in the management command?